src/CSF.SRDashboard.Client/PageValidators/SearchErrorObject.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.PageValidators
+{
+    public enum ErrorType
+    {
+        CRITERIA,
+        NO_RESULT
+    }
+    public class SearchErrorObject
+    {
+
+        public bool ErrorToggle { get; set; } = true;
```

Comment:
I suggest using a more explicit variable name like the suggestion below. 
```suggestion
        public bool IsErrorHidden { get; set; } = true;
```